driver/src/contracts/stablex_contract.rs

```diff
@@ -98,7 +98,11 @@ impl StableXContract for StableXContractImpl {
                     token_ids_for_price,
                 ),
                 account,
-                Options::default(),
+                Options::with(|mut opt| {
+                    // usual gas estimate is not working
```

Comment:
What was the problem with the gas here?

Reply:
It ran out of gas. I'm not sure if web3 estimation is broken or it always uses a fix estimate. Maybe something to look into.